pkgs/json_rpc_2/lib/src/client.dart

```diff
@@ -244,3 +254,9 @@ class _Request {
 
   _Request(this.method, this.completer, this.chain);
 }
+
+/// The default ID generator, returns an auto incrementing integer.
+int Function() get _incrementingIdGenerator {
```

Comment:
Perhaps `_createIncrementingIdGenerator()`? w/ the way its named currently, and the fact that it's a getter, it seems more like it would return an existing generator instead of creating a new one.

Reply:
done, renamed and made it a function